init.ts

```diff
@@ -536,6 +536,8 @@ try {
 const config = {
   lock: false,
   tasks: {
+    check:
+      "deno fmt --check && deno lint && deno check **/*.ts && deno check **/*.tsx",
```

Comment:
Ah just realized, we might want to wrap the globs in single quotes `'` to be compatible with all shells. I wonder if something like `'**/*.{[tj]sx?}'` would work to check all files in one go.

Reply:
Cool idea. Sadly:
```
deno task check
Task check deno fmt --check && deno lint && deno check '**/*.{[tj]sx?}'
Checked 16 files
Checked 12 files
error: Module not found "file:///Users/reed/code/temp/1454/**/*.%7B[tj]sx%3F%7D".
```

And with a simpler version:
```
deno task check
Task check deno fmt --check && deno lint && deno check '**/*.{ts,tsx}'
Checked 16 files
Checked 12 files
error: Module not found "file:///Users/reed/code/temp/1454/**/*.%7Bts,tsx%7D".
```

Even simpler:
```
deno task check
Task check deno fmt --check && deno lint && deno check '**/*.ts' && deno check '**/*.tsx'
Checked 16 files
Checked 12 files
error: Module not found "file:///Users/reed/code/temp/1454/**/*.ts".
```

I would just keep it as is for now, and if we want to get fancier, then we could build on #1422 which introduces a small cli script we can add to. Then we could do the fanciest of checks and just invoke it via `deno task cli myUltraFancyCheck` and not have to worry about different shells and globs and all of this. Just nice controllable typescript.

Reply:
I'd suggest keeping the checks simple. The checks in this PR do the job great! 🙂